src/coreclr/System.Private.CoreLib/src/System/StubHelpers.cs

```diff
@@ -382,7 +382,7 @@ internal static void ClearNative(IntPtr pNative)
         {
             if (IntPtr.Zero != pNative)
             {
-                Marshal.FreeCoTaskMem((IntPtr)(((long)pNative) - sizeof(uint)));
+                Marshal.FreeCoTaskMem(checked((IntPtr)(((long)pNative) - sizeof(uint))));
```

Comment:
This is a case where we could just `pNative - sizeof(uint)` now.
```suggestion
                Marshal.FreeCoTaskMem(pNative - sizeof(uint));
```